egs/aishell/asr1/run.sh

```diff
@@ -153,7 +157,7 @@ if [ -z ${lmtag} ]; then
 fi
 lmexpname=train_rnnlm_${backend}_${lmtag}
 lmexpdir=exp/${lmexpname}
-mkdir -p ${lmexpdir}
+# [ ! -d ${lmexpdir} ] && mkdir -p ${lmexpdir}
```

Comment:
I'm not sure your intension but you can just use the original one.
```suggestion
mkdir -p ${lmexpdir}
```